pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -101,6 +102,39 @@ def syncD2H(self, use_async=False):
         self._kokkos_state.DeviceToHost(self._state.ravel(order="C"))
         self._pre_rotated_state = self._state
 
+    def print_configuration(self, keyname=None):
+        if keyname is None:
+            print(kokkos_config()["All_Info"])
+        else:
+            if keyname not in ["Arch", "Backend", "Compiler", "Kokkos Version"]:
+                raise Exception(
+                    "Unsupported keyname. Supported keynames are: Arch, Backend, Compiler, Kokkos_Version."
+                )
+            else:
+                info_str = kokkos_config()["All_Info"]
+                info_str_list = info_str.split("\n")
+
+                keyname_out_map = {
+                    "Arch": "Architecture",
+                    "Backend": "Runtime Configuration",
+                    "Compiler": "Compiler",
+                    "Kokkos Version": "Kokkos Version",
+                }
+
+                if keyname in ["Arch", "Compiler", "Kokkos Version"]:
+                    for i in range(len(info_str_list)):
+                        if keyname_out_map[keyname] in info_str_list[i]:
+                            if keyname_out_map[keyname] == "Kokkos Version":
+                                print(info_str_list[i])
+                            else:
+                                print(info_str_list[i + 1])
+
+                if keyname is "Backend":
```

Comment:
```suggestion
                if keyname == "Backend":
```

`==` should be used here to test for equality of value.